### PR TITLE
Add support for internet password items

### PR DESCRIPTION
**Pull request by pete-woods**
I'm writing a tool to import passwords from managers like LastPass, and need to write internet passwords.

### CODE REVIEW

**Comment by joshblum** [comment]
@pete-woods can you address the CI failures?

**Comment by pete-woods** [comment]
Yes, will have a look.

**Comment by pete-woods** [comment]
@joshblum sorry it's taken so long to get back round to this, looking at fixing the linter error now

**Comment by pete-woods** [comment]
The linter is now happy on my Mac locally, so hopefully CI will like it too.

File: keychain.go [hunk 1]
```diff
@@ -265,11 +288,37 @@ func (k *Item) SetString(key string, s string) {
...
 	k.SetString(ServiceKey, s)
 }
 
+// SetServer sets the server attribute (for internet password items)
+func (k *Item) SetServer(s string) {
+	k.SetString(ServerKey, s)
+}
+
+// SetProtocol sets the protocol attribute (for internet password items)
+// Example values are: "htps", "http", "smb "
```
[the remark this answers is not in the document]
Comment (pete-woods) [reply keychain.go:302]: The space is intentional - these are all 4 characters long

**Comment by joshblum** [comment]
Thanks @pete-woods!

**Pull request merged**